lib/api-helpers/auth.ts

```diff
@@ -35,7 +35,7 @@ export const verifyEmailAuthToken = async (token: string): Promise<string> => {
 
 export const verifyAdminToken = async (
   token: string,
-  strict: boolean = true,
+  strict: boolean = true, // TODO wait why can you be not strict with this function?
```

Comment:
Ok to address your first question, we would set strict to be `false` if we just want to get a boolean of whether or not the token is an admin token rather than have the function throw an error if `!decodedToken.admin`. So for example in this function called `verifyAdminOrEmailAuthToken` we can make the check of whether the token belongs to an admin or if it belongs to an existing user without having `verifyAdminToken` throwing an error: https://github.com/hawaiians/hawaiiansintech/pull/27/files#diff-b8341c74a56328ff0ff5cb715437207f1827387d082bb722e445a5604f7d0c1bR77

`lib/auth.tsx` is updated to `lib/api-helpers/auth.ts` in the member edit PR: https://github.com/hawaiians/hawaiiansintech/pull/27/files#diff-b8341c74a56328ff0ff5cb715437207f1827387d082bb722e445a5604f7d0c1b

I'm basically adding the functions and paths that this specific PR needs that will anyways be needed for the member edit PR. Basically aligning changes as we go

Reply:
> we would set strict to be false if we just want to get a boolean of whether or not the token is an admin token
Got it. Seeing it in the other PR helps a lot.

> adding the functions and paths that this specific PR needs
Same, understanding it's related to the other PR... sounds good. 👍🏽